pcd2pgm:
  ros__parameters:
    pcd_file: /home/hyz/project/quadruped_lidar_nav2_2025_ws/src/point_lio/PCD/scans.pcd
    odom_to_lidar_odom: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    # odom_to_lidar_odom: [-0.16, 0.0, -0.18, -0.523599, 0.0, -1.5707963]
    flag_pass_through: false
    map_resolution: 0.05
    map_topic_name: map
    thre_radius: 0.1
    thre_z_max: 2.0
    thre_z_min: 0.1
    thres_point_count: 10
